src/org/opensolaris/opengrok/util/StringUtils.java

```diff
@@ -323,4 +330,20 @@ public static int countPushback(String capture, Pattern pat) {
         }
         return 0;
     }
+
+    private static int countURIUnmatchedRightParen(String value, int ridx) {
```

Comment:
this method returns either 0 or 1 and is not specific to URI (other than used for checking it) so perhaps name it `isUnmatchedRightParen` and return boolean and increment in the caller accordingly ?